Guidelines/Guidelines.md

```diff
@@ -5,3 +5,54 @@
     with no Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts.
     A copy of the license is included in the section entitled "GNU
     Free Documentation License".
+
+# PSC Event Guidelines
+
+PLLUG Summer Camp is one of the most important events. It is a challenging development marathon for PLLUG participants that lasts during summer time. Our goal is to make work on PSC projects as close to real-life software development as possible. This will ensure that participants gain important experience and learn how to be a professional developer.
```

Comment:
I think, we need to remind readers, that we will use abbreviation "PSC" in future texts instead of "PLLUG Summer Camp".